### PR TITLE
Ch/160451447/Setup an empty React Project

### PR DESCRIPTION
**Pull request by udochukwu**
**What does this PR do?**

- Setup an empty react project

**Description of Task to be completed?**

- Install Webpack, Webpack-Cli, babel, and other dependencies.

- Create a Webpack configuration file and add initial configurations.

- Configure Eslint.

**What are the relevant pivotal tracker stories?**

- https://www.pivotaltracker.com/story/show/160451447

**Screenshots (if appropriate)**

- N/A

**Questions**

- N/A

### CODE REVIEW

**Review by oladimillion — COMMENTED** [review]
LGTM

**Review by slimsolz — COMMENTED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webpack-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "watch": {
+    "start": "{src}/*.js"
```
Comment (kampkelly) [inline package.json]: This is good but let this also handle '.js' files inside sub-folders. 
You can change it to '{src}/**/*.js'

Comment (udochukwu) [reply package.json]: Alright. thanks!

**Review by onwuvic — COMMENTED** [review]
LGTM

**Review by samdiano — COMMENTED** [review]
LGTM

**Comment by kampkelly** [comment]
LGTM

**Review by s-akinrele — APPROVED** [review]
GTM

**Pull request merged**